### ISSUE TITLE
请问一下如果想在DarkFace或者Exdark上做目标检测，用您的模型做数据预处理，用您提供的哪一个数据集的预训练权重合适？还有您提供的目标检测的那个链接，是指按他的模板将您的网络模块应用上去嘛？

### ISSUE THREAD

**Issue by kairenchen123** [caiyuanhao1998/Retinexformer#20]

**Comment by caiyuanhao1998** [comment]
1. 使用LOL数据集就OK，当然也可以尝试别的权重，看暗光增强的视觉效果如何
2. 对的，按IAT的模板将我们的网络模块应用上去

**Comment by kairenchen123** [comment]
我把您的Retinexformer应用在mmdetection上，作为pre_encoder后，然后想输出增强后的图片看一下发现如下图这样，一开始我以为是检测框架的其他数据预处理的问题，然后我把那些预处理，例如Mosaic注释掉后，还是不行。请问您是否可以帮我解答一下呢？下面是我detector中的部分代码：

def extract_feat(self, batch_inputs: Tensor) -> Tuple[Tensor]:
    """Extract features.

    Args:
        batch_inputs (Tensor): Image tensor with shape (N, C, H ,W).

    Returns:
        tuple[Tensor]: Multi-level features that may have
        different resolutions.
    """
    with torch.inference_mode():
        factor = 4

        input_ = batch_inputs[0, :, :, :]
        input_ = input_ / 255.
        input_ = input_.unsqueeze(0)

        # Padding in case images are not multiples of 4
        h, w = input_.shape[2], input_.shape[3]
        H, W = ((h + factor) // factor) * \
                factor, ((w + factor) // factor) * factor
        padh = H - h if h % factor != 0 else 0
        padw = W - w if w % factor != 0 else 0
        input_ = F.pad(input_, (0, padw, 0, padh), 'reflect')

        restored = self.pre_encoder(input_)
        # Unpad images to original dimensions
        restored = restored[:, :, :h, :w]

        restored = torch.clamp(restored, 0, 1).cpu(
        ).detach().permute(0, 2, 3, 1).squeeze(0).numpy()

        cv2.imwrite('/mnt/data2/ckr/mmdetection/work_dirs/dark_out/1.png', cv2.cvtColor(img_as_ubyte(restored), cv2.COLOR_RGB2BGR))
    # 下面因为是numpy格式所以会报错，这里不用管
    x = self.backbone(restored)
    if self.with_neck:
        x = self.neck(x)
    return x

显示的图片是：
![image](https://github.com/caiyuanhao1998/Retinexformer/assets/63766314/5f19fd70-2d7a-408f-b48d-11fdef482b14)
